Prioritize/DotNet/Multi-Module/azure-pipelines_windows.yml

```diff
@@ -33,6 +33,7 @@ steps:
       Add-Content -Value "resolveAllDependencies=false" -Path ./eua.config
       Add-Content -Value "nuget.resolveDependencies=true" -Path ./eua.config
       Add-Content -Value "nuget.restoreDependencies=true" -Path ./eua.config
+      Add-Content -Value "fileSystemScan=false" -Path ./eua.config
       Add-Content -Value "includes=**/*.dll **/*.exe" -Path ./eua.config
       Add-Content -Value "excludes=**test**" -Path ./eua.config
```

Comment:
Are you looking to exclude anything (file or directory) that contains "test", or specifically the "test" directory?
If it's the latter, then it should be `**/test/**` instead of `**test**`